eZ/Bundle/EzPublishCoreBundle/ApiLoader/RepositoryConfigurationProvider.php

```diff
@@ -54,6 +51,13 @@ public function getRepositoryConfig()
         return ['alias' => $repositoryAlias] + $this->repositories[$repositoryAlias];
     }
 
+    public function pullDefaultRepository(): ?string
```

Comment:
Could you please rename it to `getDefaultRepositoryAlias()` ?    
  
Method `getCurrentRepositoryAlias` covering this logic [https://github.com/ezsystems/ezplatform-admin-ui/pull/1704/files#diff-e54466cbf9baa7083eef49e3874011f88bc048e74724640edfa77685dab2c527R55](https://github.com/ezsystems/ezplatform-admin-ui/pull/1704/files#diff-e54466cbf9baa7083eef49e3874011f88bc048e74724640edfa77685dab2c527R55) would be very useful in other contexts :wink: